docs/development/getting_started.rst

```diff
@@ -17,7 +17,7 @@ your development workstation.
 Ubuntu/Debian
 ~~~~~~~~~~~~~
 
-.. note:: Tested on: Ubuntu 14.04
+.. note:: Tested on: Ubuntu 14.04 and Debian sid (May 6th)
```

Comment:
Right - the important thing here is the "Ubuntu 14.04" means the same thing today that it meant last year, which is not true *at all* for Debian Sid. We want to avoid our docs bit-rotting as much as we can.

Adding the date to "Debian Sid" obviously fixes it in time, but that doesn't seem very useful to me. e.g. if I'm running Debian Sid 6 months from now, try to use these docs, and it doesn't work, can I use "Debian Sid (May 6th)" to compare that system to my current system in order to try to figure out what changed that broke these instructions?

If adding the date is the best we can do, then that's fine. I don't know if there is a more precise/"useful for future reference" versioning scheme used by Debian. Using the date does not seem very helpful to me (according to that thought experiment I just described), but if I'm wrong then that's fine 😁